plugins/python/src/test/kotlin/io/paddle/installTask/InstallTaskContainerTest.kt

```diff
@@ -0,0 +1,78 @@
+package io.paddle.installTask
+
+import io.paddle.plugin.python.dependencies.interpreter.InterpreterVersion
+import io.paddle.plugin.python.extensions.globalInterpreter
+import io.paddle.project.PaddleProjectProvider
+import io.paddle.testExecutor.AbstractTestContainerTest
+import org.junit.jupiter.api.*
+import org.testcontainers.junit.jupiter.Testcontainers
+import java.io.File
+import kotlin.test.assertEquals
+import kotlin.test.assertNotNull
+
+@Testcontainers
+class InstallTaskContainerTest :
+    AbstractTestContainerTest("python:3.10-buster" /* TODO: change me. A container should have c/cpp compiler & make tools to build python */) {
+    private lateinit var rootDir: File
+
+    @AfterEach
+    fun cleanupResources() {
+        rootDir.resolve(".venv").deleteRecursively()
+        rootDir.resolve(".paddle").deleteRecursively()
+    }
+
+    @Test
+    fun `minimal project resolving`() {
+        rootDir = resources.resolve("minimal-project")
+        val projectProvider = PaddleProjectProvider.getInstance(rootDir)
+        val project = projectProvider.getProject(rootDir)
+        assertNotNull(project)
+
+        assertEquals("minimal-project", project.config.get<String>("project"))
+        assertNotNull(project.tasks.get("install"))
+    }
+
+    @Test
+    fun `install task local successful`() {
+        rootDir = resources.resolve("minimal-project")
+        val projectProvider = PaddleProjectProvider.getInstance(rootDir)
+        val project = projectProvider.getProject(rootDir)
+        assertNotNull(project)
+        project.executor = executor
+
+        project.execute("install")
+        assert(project.globalInterpreter.cachedVersions.isEmpty())
+        assert(project.globalInterpreter.pythonVersion.matches(InterpreterVersion("3.10")))
+    }
+
+    @Test
+    @Order(1)
+    fun `install task install successful`() {
+        rootDir = resources.resolve("minimal-project-3.9")
+        val projectProvider = PaddleProjectProvider.getInstance(rootDir)
+        val project = projectProvider.getProject(rootDir)
+        assertNotNull(project)
+        project.executor = executor
+        assertEquals(0, project.globalInterpreter.cachedVersions.size)
+
+        project.execute("install")
```

Comment:
I suppose you should use `resolveInterpeter` task here, not `install`, since it is not the task you are really testing here, isn't it?